presto-product-tests/conf/tempto/tempto-configuration-for-docker-default.yaml

```diff
@@ -4,3 +4,4 @@ databases:
   presto:
     host: presto-master
     configured_hdfs_user: hive
+    cli_server_address: http://${databases.presto.host}:8080
```

Comment:
why do you need to add it here, when there is already an entry in tempto-configuration-for-docker-kerberos.yaml?